dss/api/bundles/__init__.py

```diff
@@ -77,6 +77,13 @@ def put(uuid: str, replica: str, json_request_body: dict, version: str = None):
                     file_metadata = handle.get(bucket, metadata_key)
                 except BlobNotFoundError:
                     continue
+                file_metadata_dict = json.loads(file_metadata)
```

Comment:
why are we parsing this twice?  it's already done on L80.

Reply:
Because the json parser becomes stronger with exercise?

fixed.